core/service/app.go

```diff
@@ -62,6 +66,118 @@ func (a App) FindAppByID(ct context.Context, appID uint64) (entity.App, *errs.Er
 	return a.appDao.FindAppByID(ct, appID)
 }
 
+func (a App) FindSecretsByAppID(ct context.Context, appID uint64) ([]entity.AppSecret, *errs.Error) {
+	return a.appSecretDao.FindSecretsByAppID(ct, appID)
+}
+
+func (a App) CreateAppSecret(ct context.Context, appID uint64, name string) (entity.AppSecret, *errs.Error) {
+	userID, ok := ctx.UserIDFromContext(ct)
+	if !ok {
+		return entity.AppSecret{}, errs.NewError(errs.Unauthenticated, "user ID not found")
+	}
+
+	genAppSecretIDReq := &proto.GenerateUniqueNumberRequest{SequenceName: "appSecretID"}
+	genAppSecretIDRes, rpcErr := a.cloudClientRegistry.GeneratorClient().GenerateUniqueNumber(ct, genAppSecretIDReq)
+	if rpcErr != nil {
+		return entity.AppSecret{}, errs.FromGRPCErr(rpcErr)
+	}
+
+	appSecret := entity.AppSecret{
+		ID:            genAppSecretIDRes.UniqueNumber,
+		Name:          name,
+		AppID:         appID,
+		AddedAt:       time.Now().UTC(),
+		AddedByUserID: userID,
+	}
+	return a.appSecretDao.CreateAppSecret(ct, appSecret)
+}
+
+func (a App) UpdateAppSecret(ct context.Context, appID uint64, appSecretID uint64, name string) (entity.AppSecret, *errs.Error) {
```

Comment:
```suggestion
func (a App) UpdateAppSecret(ct context.Context, appID uint64, input UpdateAppSecretInput) (entity.AppSecret, *errs.Error) {
```

Define input type so that we can easily add more fields in the future without changing API signature.